actioncable/lib/action_cable/connection/message_buffer.rb

```diff
@@ -30,7 +30,7 @@ def process!
 
       protected
         attr_reader :connection
-        attr_accessor :buffered_messages
+        attr_reader :buffered_messages
```

Comment:
Did that, actually, but I put it back because all the other attrs are multiline.